packages/wp-source/package.json

```diff
@@ -27,13 +27,13 @@
     "@types/jest": "^24.0.11",
     "frontity": "^1.2.2",
     "jest": "^24.7.1",
+    "node-fetch": "^2.6.0",
```

Comment:
This shouldn't be in devDependencies or it won't be installed when `wp-source` is installed as a dependency.

Reply:
Well, `node-fetch` is only needed for tests, and for the other cases `wp-source` uses `fetch` from `frontity` package. That's the reason I left `node-fetch` in `devDependencies`.

Reply:
Ohh, absolutely my bad. But then, it should be added in the `frontity` dependencies, right?

Reply:
@luisherranz yeah, you are right. I has been there for a while, though 😅

https://github.com/frontity/frontity/blob/bd8c6dc3d9872f1780e395c952737c867c88c0e8/packages/frontity/package.json#L48

If there's nothing else to change we can resolve this and merge the PR.

Reply:
Nothing else on my side! Thanks @DAreRodz.